client/e2e/utils/helpers.ts

```diff
@@ -303,3 +309,90 @@ export async function setupTestEnvironment(
   expect(await response.text()).toBe(MessageTextResponse.SETUP_SUCCESS);
   expect(response.status()).toBe(200);
 }
+
+export async function sortTableByColumnLabel(
+  page: Page,
+  columnLabel: string,
+  sortedCellTextContent: string,
+  expectedSortDirection: "ascending" | "descending" | "none" = "ascending",
+) {
+  const header = page.getByRole("columnheader").getByText(columnLabel);
+
+  const colIndex = await page
+    .locator(`[aria-label="${columnLabel}"]`)
+    .getAttribute("aria-colindex");
+  await header.click();
+
+  // wait for aria-sort to be set to ascending
+  const ariaSort = await page
+    .locator(`[aria-label="${columnLabel}"]`)
+    .getAttribute("aria-sort");
+
+  expect(ariaSort).toBe(expectedSortDirection);
+
+  // wait for response to complete
+  await page.waitForResponse((response) => response.status() === 200);
+
+  // TODO: Find a better way to wait for the table to be sorted
+  await page.waitForTimeout(1000);
```

Comment:
I spent a lot of time trying alternatives to this timeout (same with the filter helper) and came up with nothing 😞  Would love a suggestion on how to get rid of it, though for now I am stuck and left this TODO here.

Without it the assertion at the bottom of the function fails as it returns the text that exists in the first cell before the sort finishes.